readme.md

```diff
@@ -156,6 +156,22 @@ Automatic index compaction follows this algorithm:
 Replication script affects only worker pods. In each start executes schema below:
 ![Replication workflow](replication.jpg)
 
+### My cluster was down and any `Workers` can't start?
+This case is possible if all nodes where pods deployed were down (like all workers were on the same node).
+After restarting the replication script can't find the oldest node to call `CREATE CLUSTER`.
+
+To restore cluster
+1. Scale down `worker` to 0 replicas
+2. Wait until you will have no replicas in `worker` statefulset
+3. Scale up to previous replicas count
+4. That's all
+
+Pay attention, this is most short and convenient way to restore cluster. But there are some underwater rocks here
+1) If you where write through worker service as described below - it is no way to understand which node had 
+the most up-to-date information before cluster was down
+2) Accordingly, the proposed algorithm has the risk of losing data recorded during a fall
+3) So longer the interval between node drops, the greater the data loss. If all nodes fell at the same time, then the risk is minimal
```

Comment:
```suggestion
### What do I do if all workers get down **without graceful shutdown**?
It's unlikely, but possible that all workers suddenly get down (for example due to a power issue in the data center, OOM or when all the pods are on the same node). In this case when they are back up the replication script can't find the oldest node to call `CREATE CLUSTER`.

Here's what you can do to restore the cluster:
1. Scale `worker` down to 0 replicas
2. Wait until you have no replicas in the `worker` statefulset
3. Scale it up to the previous replicas count

Pay attention, this is the easiest and the most convenient way to restore the cluster in this case, but there are some things you need to be aware of:
1) If you were using the worker service for writing and all your nodes went off at about the same time there may be no way to understand which replica was 
the most up-to-date at the moment of the shutdown.
2) So there's a risk of losing data recorded during the shutdown.
3) The longer was the interval between hard shutdowns of the worker pods, the greater is the chance to lose data, since it's possible that the pod which was crashed first will form the new cluster, then the other pods that have fresher documents will just become it's replicas and the newer documents will be lost.
4) If all the nodes were shut down at the same time the risk is minimal.
```